core/app/components/LastVisitedCommunity/constants.ts

```diff
@@ -0,0 +1 @@
+export const LAST_VISITED_COOKIE = "lastVisitedCommunitySlug";
```

Comment:
i'd add a max age of 30 days to this, could maybe be longer


```suggestion
export const LAST_VISITED_COOKIE = "lastVisitedCommunitySlug";
export const LAST_VISITED_COOKIE_MAX_AGE = 60 * 60 * 24 * 30;
```